### ISSUE TITLE
Feature request to show help output inline

### ISSUE THREAD

**Issue by tbrodbeck** [google/python-fire#286]
It would be great if the output help command would not cover the whole screen and after that disappear, but if it would just append its output to the command line like most unix commands do output their help info.
You could also add this as an option, in case users need the current functionality.

**Comment by shangxiao** [comment]
Yes I'd prefer the current man page style output to be add to man and have an inline output as the default. I'm not sure how it could install the man page though, perhaps it'd be something that could be done via setuptools?

**Comment by dbieber** [comment]
As a workaround for now, you can pipe the help to cat to have it appear inline.
e.g. `python example.py -h | cat`

**Comment by nfultz** [comment]
"Short help" vs "long help" is a really good feature idea, especially for large classes, but it's not clear to me how to specifically implement it.

**Comment by Spriithy** [comment]
In a broader scope, it would be really beneficial to let people customize/tweak some settings ON/OFF etc. I'm currently using Fire to help execute REPL commands parsed with shlex, and I would really like to customize help message output (not a pager, default stdout). I cannot really think of a usecase where you would want paged help messages for a CLI tool ?

Are there any plans for supporting this in the *near* future ?
Thanks !

**Comment by dbieber** [comment]
No plans to add those configs in the next couple of months. Will revisit in the new year.
